ethpm_types/abi.py

```diff
@@ -97,7 +97,20 @@ class BaseABI(BaseModel):
     ...
 
 
-class ConstructorABI(BaseABI):
+class HashableABI(BaseABI):
```

Comment:
Would this be an issue in `BaseABI`? Is the class heirarchy import to maintain?

As far as I see, seems like only `UnprocessABI` is affected

Reply:
> As far as I see, seems like only UnprocessABI is affected

`FallbackABI` and `ReceiceABI` have no defined a `selector()` either